pandas/core/frame.py

```diff
@@ -8394,6 +8394,124 @@ def isin(self, values):
                 self.columns,
             )
 
+    def value_counts(
+        self, normalize=False, sort=True, ascending=False, bins=None, dropna=True
+    ):
+        """
+        .. versionadded:: 1.0.0
```

Comment:
I think these typically go after the 1-line description. Can you check other methods?